src/bindings/python/__init__.py

```diff
@@ -2,4 +2,4 @@
 import sys
 
 if sys.platform == "win32":
-    os.add_dll_directory(r"C:\Program Files (x86)\Common Files\Intel\Shared Libraries\bin")
+    os.add_dll_directory(r"C:\Program Files (x86)\Intel\oneAPI\compiler\2024.2\bin")
```

Comment:
I wouldn't make this change, this is compiler specific.  I think there is more work to be done here in setting this path for people who are developing the library as opposed to people who are using it.

Reply:
Agreed. However, this flags an important issue that I cannot resolve. I am effectively a "user" on windows, having never programmed in this OS until a few weeks ago. I could not build  reprosim to work without changing this line. After installing compilers out of the box I got the error that C:\Program Files (x86)\Common Files\Intel\Shared Libraries\bin did not exist, so to me both what was there to start with and what I have done (as a newbie) seem "hard coded" in some way. Those building to use would presumably need instructions to get files into Common Files or the need in some cases to change this line to ensure a successful build? I'm not sure which, or what issue to raise to try to resolve this.

Reply:
There is some documentation needed to tell people to install the redistributable package to install the required libraries in the common files directory.  The script for building the bindings should I think also be smarter about setting this path, at the moment of course it is hard-coded but it probably should be a configured path to the environment.  The common files path should only be used for the packaged version of the bindings that you would upload to PyPI.org for pip installing.